### PR TITLE
 [MIG][WIP][12.0] helpdesk_stock

### PR DESCRIPTION
**Pull request by murtuzasaleh**
WIP because of a stock_request module is in Review state.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can you please review this PR, I have changed code based on the stock_request module as per mentioned in the wiki (Migrate to use stock_request instead of helpdesk.ticket.line)

File: helpdesk_stock/views/helpdesk_views.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                                   attrs="{'invisible': [('state', '=', 'draft')]}"/>
+                            <field name="product_uom_id"
+                                   attrs="{'readonly': [('state', 'in', ('sale', 'done', 'cancel'))]}"
+                                   options="{'no_open': True, 'no_create': True}"
+                                   groups="uom.group_uom"/>
+                            <field name="state" readonly="1"/>
+                        </tree>
+                    </field>
+                </page> -->
+                <page string="Stock Requests">
```
Comment (max3903) [inline helpdesk_stock/views/helpdesk_views.xml]: @murtuzasaleh Please delete the commented lines and rename this page "Inventory". Thank you.

File: oca_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+stock-logistics-warehouse
+
```
Comment (max3903) [inline oca_dependencies.txt]: Remove empty line.

**Review by max3903 — APPROVED** [review]

**Comment by murtuzasaleh** [comment]
@osi-scampbell can you please review this PR.

**Review by osi-scampbell — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**